mne/io/edf/tests/test_edf.py

```diff
@@ -525,3 +526,19 @@ def test_exclude():
     raw = read_raw_edf(edf_path, exclude="I[1-4]")
     for ch in exclude:
         assert ch not in raw.ch_names
+
+
+@testing.requires_testing_data
+def test_ch_types():
+    """Test reading of channel types from EDF channel label."""
+    raw = read_raw_edf(edf_chtypes_path)
+
+    # get the channel types for all channels
+    ch_types = raw.get_channel_types()
+    assert all([x in ch_types for x in ['eeg', 'ecg']])
```

Comment:
```suggestion
    assert all(x in ch_types for x in ['eeg', 'ecg'])
```
`all` can handle generators